x-pack/plugins/apm/server/lib/helpers/convert_ui_filters/get_ui_filters_es.ts

```diff
@@ -27,22 +27,19 @@ export function getUiFiltersES(uiFilters: UIFilters) {
       };
     }) as ESFilter[];
 
-  // remove undefined items from list
   const esFilters = [
-    getKueryUiFilterES(uiFilters.kuery),
-    getEnvironmentUiFilterES(uiFilters.environment),
-  ]
-    .filter((filter) => !!filter)
-    .concat(mappedFilters) as ESFilter[];
+    ...getKueryUiFilterES(uiFilters.kuery),
+    ...getEnvironmentUiFilterES(uiFilters.environment),
+  ].concat(mappedFilters) as ESFilter[];
```

Comment:
Nice change, thanks! Can we use spread for `mappedFilters` instead of concat (for consistency)

```suggestion
    ...getKueryUiFilterES(uiFilters.kuery),
    ...getEnvironmentUiFilterES(uiFilters.environment),
    ...mappedFilters
  ] as ESFilter[];
```